locales/de.json

```diff
@@ -1,574 +1,911 @@
+      "help": "ARIA-progressbar Knoten müssen einen barrierefreien Namen besitzen."
+    },
+    "aria-required-attr": {
+      "description": "",
+      "help": "Erforderliche ARIA-Attribute müssen bereitgestellt werden."
+    },
+    "aria-required-children": {
+      "description": "",
+      "help": "Bestimmte ARIA-Rollen müssen spezifische, untergeordnete Kind-Rollen enthalten."
+    },
+    "aria-required-parent": {
+      "description": "",
+      "help": "Bestimmte ARIA-Rollen müssen in spezifischen, übergeordneten Eltern-Rollen enthalten sein."
+    },
+    "aria-roledescription": {
+      "description": "Stellt sicher, dass ARIA-roledescription nur im Zusammenhang mit einer im- oder expliziten Rolle verwendet wird.",
+      "help": "Nutze aria-roledescription für Elemente mit einer semantischen Rolle."
+    },
+    "aria-roles": {
+      "description": "",
+      "help": "Verwendete ARIA-Rollen müssen gültigen Werten entsprechen."
+    },
+    "aria-toggle-field-name": {
+      "description": "Stellt sicher, dass jedes ARIA-toggle-Feld ein barrierefreien Namen besitzt.",
+      "help": "ARIA-toggle-Felder besitzen einen barrierefreien Namen."
+    },
+    "aria-tooltip-name": {
+      "description": "Stellt sicher, dass jeder ARIA-tooltip Knoten einen barrierefreien Namen besitzt.",
+      "help": "ARIA-tooltip Knoten benötigen einen barrierefreien Namen."
+    },
+    "aria-treeitem-name": {
+      "description": "Stellt sicher, dass jeder ARIA-treeitem Knoten einen barrierefreien Namen besitzt.",
+      "help": "ARIA-treeitem Knoten benötigen einen barrierefreien Namen."
+    },
+    "aria-valid-attr-value": {
+      "description": "",
+      "help": "Verwendete ARIA-Attribute müssen gültigen Werten entsprechen."
+    },
+    "aria-valid-attr": {
+      "description": "",
+      "help": "Verwendete ARIA-Attribute müssen gültigen Namen entsprechen."
+    },
+    "audio-caption": {
+      "description": "",
+      "help": "<audio>-Elemente müssen eine Untertitelung (captions track) besitzen."
+    },
+    "autocomplete-valid": {
+      "description": "Stellt sicher, dass das autocomplete-Attribut korrekt ist und für das form-Feld geeignet ist.",
+      "help": "autocomplete-Attribute müssen korrekt genutzt werden."
+    },
+    "avoid-inline-spacing": {
+      "description": "Stellt sicher, dass der Zeichenabstand durch benutzerdefinierte Stylesheets angepasst werden kann.",
+      "help": "Zeichenabstände müssen durch benutzerdefinierte Stylesheets anpassbar sein."
+    },
+    "blink": {
+      "description": "",
+      "help": "<blink>-Elemente sind veraltet und dürfen nicht verwendet werden."
+    },
+    "button-name": {
+      "description": "",
+      "help": "Schaltflächen müssen wahrnehmbaren Text enthalten."
+    },
+    "bypass": {
+      "description": "",
+      "help": "Die Seite muss Wege bereitstellen, um mehrfache Blöcke zu übergehen."
```

Comment:
would `Die Seite muss Wege bereitstellen, um mehrfach-Blöcke zu überspringen.` be more accurate? `repeated blocks` are blocks that appear on more than one page.

Also seems to me like ueberspringen is more accurate - wdyt?

Reply:
You are right. I changed it to a better wording.